workshops/models.py

```diff
@@ -329,6 +336,34 @@ def __str__(self):
     def get_absolute_url(self):
         return reverse('event_details', args=[self.get_ident()])
 
+    def get_repository_url(self):
+        """Return self.url formatted as it was repository URL.
+
+        Repository URL is: https://github.com/user/repository/
```

Comment:
"Repository URL is as specified in REPO_FORMAT" (so that the doc string doesn't go stale).